project1/css/index.css

```diff
@@ -16,7 +16,7 @@ article, aside, canvas, details, embed,
 figure, figcaption, footer, header, hgroup, 
 menu, nav, output, ruby, section, summary,
 time, mark, audio, video {
-	margin: 15px;
+	margin: 5px 20px;
```

Comment:
By doing this you add a margin to all of those items. This caused your menu to be a bit far separated.

![image](https://user-images.githubusercontent.com/12449668/37110666-c6b5cae4-21fa-11e8-9937-b6ce404d8268.png)